GlobalConfigurations/AppConfigurations.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json;
+using GlobalConfigurations.Models;
+
+namespace GlobalConfigurations
+{
+    public class AppConfigurations
```

Comment:
I see this used across multiple projects, since config file cannot be changed on runtime, you can just read it once  from the json file and use it in the whole program lifetime. So you can declare a static object of this class in program.cs of each project, and then pass it to the class constructors to be used.

`AnomaliesDetection` would need to be changed to a class with a constructor that accepts this object for example.